views/result.erb

```diff
@@ -0,0 +1,21 @@
+<body>
+  <div>
+
+  <% if @rock %>
+    <h2> You have chosen <%= @rock %> and Computer has chosen <%= @computer %> </h2>
+    <h1><%= @result.play(@rock, @computer) %></h1>
```

Comment:
This line would be better with some CSS added to make it larger and the H1 tag changed to something else I guess